### PR TITLE
feature/#2948-pop-ups-for-ubs-client-profile

### PR DESCRIPTION
**Pull request by rostikdoroshenko**
created modal window 'change password';
created modal window 'delete profile';
added base styles.
change-password
delete-profile

### CODE REVIEW

**Review by Anastasiia-Hevchuk — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts [hunk 1]
```diff
@@ -43,6 +48,20 @@ export class UbsClientProfilePageComponent implements OnInit {
...
+      hasBackdrop: true
+    });
+    dialogRef.afterClosed();
+  }
+
+  public openChangePasswordDialog() {
+    const dialogRef = this.dialog.open(UbsProfileChangePasswordPopUpComponent, {
+      hasBackdrop: true
+    });
+    dialogRef.afterClosed();
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts:62]: afterClosed returns observables, so do we need this here ?

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts [hunk 2]
```diff
@@ -43,6 +48,20 @@ export class UbsClientProfilePageComponent implements OnInit {
     });
   }
 
+  public openDeleteProfileDialog() {
+    const dialogRef = this.dialog.open(UbsProfileDeletePopUpComponent, {
+      hasBackdrop: true
+    });
+    dialogRef.afterClosed();
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts:55]: afterClosed returns observables, so do we need this here ?

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts (hunk 1 above)
Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts]: fixed

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts (hunk 2 above)
Comment (rostikdoroshenko) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts]: fixed

**Review by tkhristina — APPROVED** [review]

**Pull request merged**